extensions/Worker.Extensions.Tables/src/Worker.Extensions.Tables.csproj

```diff
@@ -18,8 +18,8 @@
   
   <ItemGroup>
     <PackageReference Include="Azure.Data.Tables" Version="12.8.3" />
-    <PackageReference Include="Microsoft.Extensions.Azure" Version="1.7.5" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Azure" Version="1.12.0" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
```

Comment:
Please note this is a major version change for this dependency, brought in by the new version of `Microsoft.Extensions.Azure`.

Reply:
This change is fine.

Reply:
This is fine.